runtime/basilisk/src/governance.rs

```diff
@@ -149,7 +154,9 @@ impl pallet_collective::Config<CouncilCollective> for Runtime {
 	type MaxProposals = CouncilMaxProposals;
 	type MaxMembers = CouncilMaxMembers;
 	type DefaultVote = pallet_collective::PrimeDefaultVote;
-	type WeightInfo = ();
+	type WeightInfo = (); // TODO
```

Comment:
this definetly should be done

Reply:
btw, there is more pallets that are configured to use default weights